### DISCUSSION TITLE
Message field (textarea) not coming through on email submissions

### DISCUSSION THREAD

**Discussion by FAN-ian-taylor** [statamic/cms#6853] [Troubleshooting] [answered]
Hi there, I have a strange issue where the **message** field doesn't come through on sent emails but is showing on the submissions. I can't seem to find a reason for it in the Statamic docs. My antlers template is as follows:

`<p>From: {{ first_name }} {{ last_name }}</p>
<p>Email: {{ work_email }}</p>
<p>Message:</p>
<p>{{ message }}</p>`

The other fields come through fine, I expect the issue is something related to textarea handling as the other fields are inputs and come through fine. I'm sure I'm missing something but has anyone come across similar? Or know a way to solve this problem?

**Comment by jasonvarga** [comment, marked as answer]
`message` is a reserved word, unfortunately. Try picking a different name. Like `content`, for example.


https://statamic.dev/forms#the-blueprint

![CleanShot 2022-10-10 at 10 31 47](https://user-images.githubusercontent.com/105211/194890106-9a8ca6cd-83bc-44d6-a543-88c671711336.png)

**Reply by FAN-ian-taylor** [reply]
Thanks @jasonvarga you're a life saver! Not sure how I've missed that
